frontend/src/scenes/insights/InsightPageHeader.tsx

```diff
@@ -246,7 +256,18 @@ export function InsightPageHeader({ insightLogicProps }: { insightLogicProps: In
                             </LemonButton>
                         )}
                         {insightMode !== ItemMode.Edit && hasDashboardItemId && (
-                            <AddToDashboard insight={insight} setOpenModal={setAddToDashboardModalOpenModal} />
+                            <>
+                                <NotebookSelectButton
```

Comment:
I think this is fine "for now" but probably we want to come up with some combined button for adding an insight to either a dashboard or a notebook as otherwise that header could become a bit cramped